### PR TITLE
Added baritone :)

### PR DESCRIPTION
**Pull request by timoreo22**
Yep baritone added in

This is maked as a draft bc https://github.com/MeteorDevelopment/baritone needs to be updated to [wagyourtail's fork](https://github.com/wagyourtail/baritone/tree/1.17)

Once that is done, a small tweak and baritone will be ready to go :)

### CODE REVIEW

**Comment by Intoprelised** [comment]
it already has baritone

**Comment by timoreo22** [comment]
> 
> 
> it already has baritone

not on 1.17.1 :)

File: build.gradle [hunk 1]
```diff
@@ -51,6 +52,12 @@ dependencies {
 	}
 	include "com.github.MeteorDevelopment:baritone:${project.baritone_version}"*/
 
+    // Baritone, as a random jar file, best thing ever
+    modImplementation "wagyourtail:baritone:1.7.0"
+    //modImplementation (name : "baritone-but-the-fucking-cache-is-annoying")
```
Comment (C10udburst) [inline build.gradle:57]: Remove da comment

Comment (timoreo22) [reply build.gradle]: fixed

**Comment by AstroOrbis** [comment]
w a t

**Comment by AstroOrbis** [comment]
it legit already has baritone on all versions

**Comment by RedCarlos26** [comment]
Not in the 1.17 dev builds @AstroOrbis

**Comment by AstroOrbis** [comment]
They are called :sparkles: dev builds :sparkles for a reason

**Comment by timoreo22** [comment]
Should be ready to go as soon as https://github.com/MeteorDevelopment/baritone/pull/1 is merged

**Comment by timoreo22** [comment]
Should be ready to merge now !

**Comment by AstroOrbis** [comment]
aight well i was wrong, thanks for this

**Pull request merged**